qutebrowser/mainwindow/tabwidget.py

```diff
@@ -394,6 +416,7 @@ def _on_config_changed(self, option: str):
                       "tabs.indicator.width",
                       "tabs.min_width",
                       "tabs.pinned.shrink"]:
+            self._pinned_statistics.cache_clear()
```

Comment:
Nitpick: Use the same order (before/after `_minimum_tab_size_hint_helper.cache_clear`) like below - doesn't matter which one :wink: